Pepperdash Core/Pepperdash Core/Comm/GenericTcpIpClient.cs

```diff
@@ -388,8 +388,8 @@ public void SendEscapedText(string text)
         /// <param name="bytes"></param>
 		public void SendBytes(byte[] bytes)
 		{
-			//if (Debug.Level == 2)
-			//    Debug.Console(2, this, "Sending {0} bytes: '{1}'", bytes.Length, ComTextHelper.GetEscapedText(bytes));
+			
+			Debug.Console(2, this, "Sending {0} bytes: '{1}'", bytes.Length, ComTextHelper.GetEscapedText(bytes));

```

Comment:
This adds back in the debug statements that print data about to be sent. These can’t be included in this PR. There’s already a PR that adds the new StreamDebugging mechanism merged earlier today.